.github/docker/Dockerfile.centreon-web-22.10.0-alma-8

```diff
@@ -0,0 +1,68 @@
+FROM docker.centreon.com/proxy/almalinux:8 AS web_fresh
+
+RUN \
+
+dnf install -y https://rpms.remirepo.net/enterprise/remi-release-8.rpm && \
+dnf module reset -y php && \
+dnf module install -y php:remi-8.1 && \
+
+dnf install -y dnf-plugins-core && \
+
+dnf install -y epel-release && \
+
+dnf install -y yum-utils && \
+dnf config-manager --set-enabled powertools && \
+
+curl -LsS https://r.mariadb.com/downloads/mariadb_repo_setup | bash -s -- --os-type=rhel --skip-check-installed --os-version=8 --mariadb-server-version="mariadb-10.5" && \
+dnf install -y mariadb-server mariadb && \
+echo "[server] \
+skip-grant-tables \
+log_output=FILE \
+general_log_file=/var/lib/mysql/queries.log \
+general_log=0 \
+slow_query_log_file=/var/lib/mysql/slow_queries.log \
+slow_query_log=1 \
+innodb_file_per_table \
+innodb_flush_method=O_DIRECT \
+innodb_log_file_size=4M \
+innodb_fast_shutdown=0 \
+" > /etc/my.cnf.d/container.cnf && \
+
+
+dnf config-manager --add-repo https://yum.centreon.com/standard/22.10/el8/stable/noarch && \
+dnf config-manager --add-repo https://yum.centreon.com/standard/22.10/el8/stable/x86_64 && \
```

Comment:
does it work ? install rpm release directly as it's described in docs.centreon.com